### ISSUE TITLE
Modqueue notifications from new modmail link to wrong domain

### ISSUE THREAD

**Issue by leviroth** [creesch/reddit-moderator-toolbox#789]
To reproduce:

1. Have the new modmail as your only reddit tab open
2. Wait for, or generate, a new modqueue item
3. Click the "New modqueue item" notification
4. The link goes to https://mod.reddit.com/r/mod/about/modqueue, which is a 404 error.

I found this on Firefox. I don't know if it applies to other kinds of notification.

**Comment by Geo1088** [comment]
>I don't know if it applies to other kinds of notification.

I'm pretty sure it does. There must still be some relative links left from when we tried making everything absolute.

**Comment by creesch** [comment]
Well actually, it shouldn't show notifications on new modmail. Unless we changed that in the last release or they changed their settings. 

We prevent toolbox from doing so in subdomains to prevent double notifications.

**Comment by Geo1088** [comment]
@creesch I'm not clear on happened with that setting, but I thought it was made so that mod.reddit was considered the same as www.reddit for notifications.

**Comment by leviroth** [comment]
There are some URLs in the notifications module that don't include `TBUtils.baseDomain`. I could add it, if that would be the best way to fix.

**Comment by creesch** [comment]
No, they shouldn't be blindly added. Some of them specifically don't use it.

**Comment by leviroth** [comment]
What about specifically the URLs that are passed as the `url` parameter of `TBUtils.notification`?

**Comment by leviroth** [comment]
>Well actually, it shouldn't show notifications on new modmail. Unless we changed that in the last release or they changed their settings.

I just found the setting that you're talking about. For some reason I had that disabled.

**Comment by Geo1088** [comment]
Is this worth revisiting now that we have a separate version for redesign stuff?
